docs/standard/serverless-architecture/serverless-architecture/index.md

```diff
@@ -0,0 +1,74 @@
+* Extract Transform and Load (ETL)
+
+There are other, more specific use cases that are covered later in this document.
+
+### Monoliths and "starving the beast"
+
+A common challenge is migrating an existing monolithic application to the cloud. The least risky approach is to "life and shift" entirely onto virtual machines. Many shops prefer to use the migration as an opportunity to modernize their code base. A practical approach to migration is called "starving the beast." In this scenario, the monolith is migrated "as is" to start with. Then, select services are modernized. In some cases, the signature of the service is identical to the original, it simply is hosted as a function. Clients are updated to use the new service rather than the monolith endpoint. In the interim, steps such as database replication enable microservices to host their own storage even when transactions are still handled by the monolith. Eventually, clients are migrated onto the new services. The monolith is "starved" (its services no longer called) until all functionality has been replaced. The combination of serverless and proxies can facilitate much this migration.
+
+![Serverless monolith migration](./media/serverless-monolith-migration.png)
+
+To learn more about this approach, watch the video: [Bring your app to the cloud with serverless Azure Functions](https://channel9.msdn.com/Events/Connect/2017/E102).
+
+### Web apps
+
+Web apps are great candidates for serverless applications. There are two common approaches to web apps today: server-driven, and client-driven, or Single Page Application (SPA). Server-driven web apps typically use a middleware layer to issue API calls to render the web UI. SPA applications make REST API calls directly from the browser. In both scenarios, serverless can accommodate the middleware or REST API request by providing the necessary business logic. A common architecture is to stand up a lightweight static web server that serves HTML, CSS, JavaScript, and other browser assets. The web app then connects to a microservices backend.
+
+### Mobile back-ends
+
+The event-driven paradigm of serverless apps makes them ideal as mobile back-ends. The mobile device triggers the events and the serverless code executes to satisfy requests. Taking advantage of a serverless model enables developers to enhance business logic without necessarily having to deploy an application update. The serverless approach also enables teams to share endpoints and work in parallel. Perhaps one of the biggest advantages is that mobile developers can build business logic without becoming experts on the server side. Traditionally, mobile apps connected to on-premises services. Building the service layer required understanding the server platform and programming paradigm. Developers worked with operations to provision servers and configure them appropriately. Sometimes days or even weeks were spent on building a deployment pipeline.
```

Comment:
This paragraph loses its focus starting with the sentence "Perhaps one of ...."

I think the point here is that each serverless function is a small, self-contained deployment, and as such, easier to build and modify than adding features to a large back-end system. The separate components also simplify deployment.